app/actions/EventActions.ts

```diff
@@ -375,13 +370,14 @@ export function unfollow(followId: EntityId, eventId: EntityId) {
   });
 }
 
-export function isUserFollowing(eventId: EntityId) {
-  return callAPI<boolean>({
```

Comment:
I feel like the function name here is very misleading if it does not return a boolean

Reply:
It 100% is. I kind of feel like whoever wrote the funciton didn't even know what the endpoint returned. I didn't change it because I was considering adding an endpoint that did return a boolean, but I guess I can change it as I never got around to doing that.